packer/ansible/roles/images/tasks/main.yml

```diff
@@ -49,11 +49,11 @@
 
 # hacks for install CoreOS demo - to be replaced with proxy functionality
 - name: make CoreOS directory for hosting CoreOS demonstration booting
-  file: path=/home/vagrant/src/on-http/static/http/coreos/835.8.0 state=directory
+  file: path=/home/vagrant/src/on-http/static/http/coreos/835.9.0 state=directory
```

Comment:
yeah - my path out of this hell hole is when we add the proxy support, then I can remote point to the CoreOS structures and call it "good" for a demo setup. And save a couple hundred MB on the Vagrant instance as well...